### DISCUSSION TITLE
Clear cache

### DISCUSSION THREAD

**Discussion by andyburgessmd** [Budibase/budibase#8682] [Help]
Is there a way that the db cache used in an application can be cleared?
For example if I add a new row to a database table it often takes maybe 5 minutes for that row to show in a table in the application. I can force it to show by publishing the whole application but that is not ideal.
I can;t tell if the cache is client side (i.e api query to load data has an expiry) or if the query is cached in the api and the requests from the client gets the same data in the response.

**Comment by Rory-Powell** [comment]
Hey @andyburgessmd!

This is actually by design, a budibase app really consists of two separate databases. When a publish occurs, the `dev` database is replicated through to the `prod` database. This means that when you publish, the rows (and screens, components etc) you create inside the builder will appear in the published app. 

Similarly, changes created in the production database by using the app will replicated back to the development database so the two will be in sync. 

Does this explain the behaviour you are seeing?

**Reply by andyburgessmd** [reply]
It does yes thanks @Rory-Powell.
